client/app/pages/queries/components/QueryPageHeader.jsx

```diff
@@ -199,8 +199,8 @@ export default function QueryPageHeader({
 
         {!queryFlags.isNew && (
           <Dropdown overlay={moreActionsMenu} trigger={["click"]}>
-            <Button data-test="QueryPageHeaderMoreButton">
-              <EllipsisOutlinedIcon rotate={90} />
+            <Button data-test="QueryPageHeaderMoreButton" aria-label="More actions">
+              <EllipsisOutlinedIcon rotate={90} aria-label="" />
```

Comment:
May be a newbie question as I'm no expert with those, but what's `aria-label=""` used for? Why not `aria-hidden="true"`?

Reply:
The intention here is to make sure Ant's `aria-labels` will not be displayed, so as to avoid confusion. Theoretically `aria-hidden` does the trick, but I wanted to override it in some cases either way. Not a lot to back me up on this, though.

Reply:
Just checked with a real SR. It does the trick.